remote-api/src/main/java/org/alfresco/rest/api/ContentStorageInformation.java

```diff
@@ -44,34 +45,34 @@
     /**
      * Note: Currently marked as experimental and subject to change.
      *
-     * @param nodeId          Identifier of the node
+     * @param nodeRef         Node reference
      * @param contentPropName Qualified name of content property (e.g. 'cm_content')
      * @param parameters      {@link Parameters} object to get the parameters passed into the request
      * @return {@link ContentStorageInfo} object consisting of qualified name of content property and a map of storage properties
      */
     @Experimental
-    ContentStorageInfo getStorageInfo(String nodeId, String contentPropName, Parameters parameters);
+    ContentStorageInfo getStorageInfo(NodeRef nodeRef, String contentPropName, Parameters parameters);
 
     /**
      * Note: Currently marked as experimental and subject to change.
      *
-     * @param nodeId                Identifier of the node
+     * @param nodeRef               Node reference
      * @param contentPropName       Qualified name of content property (e.g. 'cm_content')
      * @param archiveContentRequest {@link ArchiveContentRequest} object holding parameters for archive content request
      * @return true when request successful, false when unsuccessful
      */
     @Experimental
-    boolean requestArchiveContent(String nodeId, String contentPropName, ArchiveContentRequest archiveContentRequest);
+    boolean requestArchiveContent(NodeRef nodeRef, String contentPropName, ArchiveContentRequest archiveContentRequest);
 
     /**
      * Note: Currently marked as experimental and subject to change.
      *
-     * @param nodeId                        Identifier of the node
+     * @param nodeRef                       Node reference
      * @param contentPropName               Qualified name of content property (e.g. 'cm_content')
      * @param restoreArchivedContentRequest {@link RestoreArchivedContentRequest} object holding parameters for restore from archive request
      * @return true when request successful, false when unsuccessful
      */
     @Experimental
-    boolean requestRestoreContentFromArchive(String nodeId, String contentPropName,
+    boolean requestRestoreContentFromArchive(NodeRef nodeRefd, String contentPropName,
```

Comment:
should this be nodeRef rather than nodeRefd?

Reply:
Good catch - will fix. I'll also update new files to be 2022 (the existing ones have been done in a separate PR).

Reply:
Done. Thanks.